### PR TITLE
[DOC]: Fix invalid links and add materials to notebooks

### PR DESCRIPTION
**Pull request by huiyuxie**
I'm new to CuGraph repository and would like to contribute further in the future. So I start with minor tasks with docs to familiarize myself with the contribution process in this repository.

Here is what I have done in this PR:
- Fixed some invalid links in the documentation.
- Corrected and added formulas related to centrality.
- Added more references to the centrality to ensure consistency.

Also, the images located at https://github.com/rapidsai/cugraph/tree/main/python/cugraph cannot be displayed because it is symbolic link to the repository's README file. But it's not a major issue.

### CODE REVIEW

**Comment by huiyuxie** [comment]
Should be tagged with the doc label.

**Review by acostadon — COMMENTED** [review]
Changes look good. This overlaps with PR #3938 and will be merged and deconflicted after that one is merged.

**Comment by acostadon** [comment]
This can me merged and deconflicted after #3938 is merged.

**Comment by huiyuxie** [comment]
Thank you all! I fixed one more place related to the formula.

File: notebooks/algorithms/centrality/Centrality.ipynb [hunk 1]
```diff
@@ -36,22 +36,34 @@
...
-    "$C_d(v) = \\frac{{\\text{{degree of vertex }} v}}{{\\text{{total number of vertices}} - 1}}$\n",
-    "\n",
+    "$C_{degree}(v) = \\frac{{\\text{degree of vertex} \\ v}}{{\\text{total number of vertices} - 1}}$\n",
     "\n",
+    "See:\n",
+    "* [Degree (graph theory) on Wikipedia](https://en.wikipedia.org/wiki/Degree_(graph_theory)) for more details on the algorithm.\n",
+    "* [Learn more about Degree Centrality](https://www.sci.unich.it/~francesc/teaching/network/degree.html)\n",
     "\n",
-    "___Closeness centrality – coming soon___ <br>\n",
+    "__Closeness Centrality__ <br>\n",
```
Comment (acostadon) [inline notebooks/algorithms/centrality/Centrality.ipynb:45]: coming soon was in here because this notebook doesn't include that algorithm,

**Review by acostadon — APPROVED** [review]
There are some nice improvements here. once merged we can check the links and change as necessary.

**Review by BradReesWork — APPROVED** [review]

**Pull request merged**